data_steward/validation/main.py

```diff
@@ -174,33 +192,50 @@ def run_validation(hpo_id, error_ignore_flag=False):
         errors = []
         results = []
         found_cdm_file_names = found_cdm_files
+
+        # Create all tables first to simplify downstream processes
+        # (e.g. ehr_union doesn't have to check if tables exist)
         for cdm_file_name in common.CDM_FILES:
+            cdm_table_name = cdm_file_name.split('.')[0]
+            table_id = bq_utils.get_table_id(hpo_id, cdm_table_name)
+            bq_utils.create_standard_table(cdm_table_name, table_id, drop_existing=True)
+
+        for cdm_file_name in common.CDM_FILES:
+            logging.info('Validating file `{file_name}`'.format(file_name=cdm_file_name))
             found = parsed = loaded = 0
             cdm_table_name = cdm_file_name.split('.')[0]
+
             if cdm_file_name in found_cdm_file_names:
                 found = 1
                 load_results = bq_utils.load_cdm_csv(hpo_id, cdm_table_name, folder_prefix)
                 load_job_id = load_results['jobReference']['jobId']
-
                 incomplete_jobs = bq_utils.wait_on_jobs([load_job_id], retry_count=BQ_LOAD_RETRY_COUNT)
 
                 if len(incomplete_jobs) == 0:
                     job_resource = bq_utils.get_job_details(job_id=load_job_id)
                     job_status = job_resource['status']
                     if 'errorResult' in job_status:
-                        error_messages = ['{}'.format(item['message'], item['location']) for item in job_status['errors']]
-                        errors.append((cdm_file_name, ' || '.join(error_messages)))
+                        # These are issues (which we report back) as opposed to internal errors
+                        issues = ['{}'.format(item['message'], item['location']) for item in job_status['errors']]
```

Comment:
(also a codacy issue) we need a '{} at {}' instead of '{}'.

Reply:
thank you will do